skins/oasis/modules/OasisController.class.php

```diff
@@ -292,28 +371,48 @@ private function loadJs() {
 			$assetGroups[] = 'oasis_anon_js';
 		}
 
+
+		$jsLoader = '';
+
 		wfRunHooks('OasisSkinAssetGroups', array(&$assetGroups));
 
 		// add groups queued via OasisController::addSkinAssetGroup
 		$assetGroups = array_merge($assetGroups, self::$skinAssetGroups);
 
-		// SUS-905: Load all AssetsManager JS groups in a single HTTP request
-		/** @var SkinOasis $skin */
-		$skin = $this->getContext()->getSkin();
-		$jsLoader = $skin->getScriptsWithCombinedGroups( $assetGroups );
+		$assets = $this->assetsManager->getURL( $assetGroups ) ;
+
+		// jQueryless version - appears only to be used by the ad-experiment at the moment.
+		// disabled - not needed atm (and skipped in wsl-version anyway)
+		// $assets[] = $this->assetsManager->getURL( $isLoggedIn ? 'oasis_nojquery_shared_js_user' : 'oasis_nojquery_shared_js_anon' );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)